### PR TITLE
Merge Brandwatch changes

### PR DESCRIPTION
**Pull request by adams85**
### Describe the purpose of your pull request

This PR merges the changes from the fork made by Brandwatch (https://github.com/brandwatch/migrate-ld-cc).

### Requirement checklist (only if applicable)

- [ ] I have covered the applied changes with automated tests.
- [ ] I have executed the full automated test set against my changes.
- [ ] I have validated my changes against all supported platform versions.
- [ ] I have read and accepted the [contribution agreement](https://github.com/configcat/legal/blob/main/contribution-agreement.md).

### CODE REVIEW

File: bin/migrate.js [hunk 1]
```diff
@@ -1,104 +1,153 @@
...
+    ffs: new ConfigCatFeatureFlags(),
+    envs: new ConfigCatEnvironments(),
+    tags: new ConfigCatTags(),
+    segments: new ConfigCatSegments(),
+};
 
 async function init() {
-  const { ldProjects, ldEnvs, ldFeatureFlags } = await getLaunchDarklyData();
-  await crudConfigCatData(ldProjects, ldEnvs, ldFeatureFlags);
+    await cleanExistingConfigCatData();
```
Comment (laliconfigcat) [inline bin/migrate.js:36]: I'm not sure if this is the right way to clean up the feature flags. Seems a bit risky to delete a feature flag that may be in use. Maybe introducing some parameter would be good to set if we should clean up feature flags before the migration.

File: data/ccEnvironments.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[
```
Comment (laliconfigcat) [inline data/ccEnvironments.json:1]: Is this file necessary?

**Review by laliconfigcat — CHANGES_REQUESTED** [review]

**Review by laliconfigcat — APPROVED** [review]

File: bin/migrate.js (hunk 1 above)
Comment (adams85) [reply bin/migrate.js:36]: I'll look into it, and make adjustments if necessary in subsequent PRs.

File: data/ccEnvironments.json (hunk 2 above)
Comment (adams85) [reply data/ccEnvironments.json:1]: I'll look into it, and make adjustments if necessary in subsequent PRs.
